libraries/dataGetters.js

```diff
@@ -57,25 +33,6 @@ function getAOE4PlayerLastGames(profileID, since) {
                                 }
                             }
                         }
-
-                        for (var [gameID, game] of Object.entries(originalData.games)) {
-                            for (var [teamID, team] of Object.entries(game.teams)) {
-                                for (var [playerID, player] of Object.entries(team)) {
-                                    getPlayMateData(player.player.profile_id, gameID, teamID, playerID).then(function (playmateData) {
-                                        let recGameID = playmateData['request']['gameID'];
-                                        let recTeamID = playmateData['request']['teamID'];
-                                        let recPlayerID = playmateData['request']['playerID'];
-                                        newData['games'][recGameID]['teams'][recTeamID][recPlayerID]['playmate'] = playmateData;
-                                        extraDataPlayers++;
-                                        if (extraDataPlayers == totalPlayers) {
-                                            return resolve(newData);
-                                        }
-                                    }, function (err) {
-                                        return resolve(originalData);
-                                    });
-                                }
-                            }
-                        }
```

Comment:
if we are removing this block of code, we should end with

`return resolve(originalData);`

in order to finish the promisse